app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt

```diff
@@ -130,8 +130,10 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
         realm.beginTransaction()
         if (which) {
             model?.isPositiveRated = what
+            Toast.makeText(contexts, R.string.rate_chat, Toast.LENGTH_SHORT).show()
         } else {
             model?.isNegativeRated = what
+            Toast.makeText(contexts, R.string.rate_chat, Toast.LENGTH_SHORT).show()
```

Comment:
```js
if (i ==1)
  print('Yes')
else
  print('Yes')
```

What's the difference?

Reply:
Sir, I am making it in one line. But, here look that it has been succesfully added to the database.